portfolio/src/main/webapp/script.js

```diff
@@ -98,7 +98,11 @@ function fetchComments() {
         var i;
         for (i = 0; i < comments.length; i++) {
             commentsListElement.appendChild(
-                createListElement(comments[i].text));
+                createListTextElement(comments[i].text));
+            if(comments[i].imageURL != null) {
```

Comment:
nit: !==

(doesn't really matter here, but in general favor !== since it won't consider things that can be converted to each other as equal